index.html

```diff
@@ -31,23 +37,26 @@ <h2>I'am a Software Developer</h2>
              you see and have a project you need coded, 
              don't hestiate to contact me.
         </p>
-        
+        </div>
+        <div class="intro-icons">
        <ul>
         <li><img src="Images/Icons/NormalButton/Tertiary/Icons/github-icon.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/Icon-Linkedin.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/hand-icon.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/NormalButton/Tertiary/Icons/twitter-icon.png" alt="soclai media icons"></li>
         <li><img src="Images/Icons/medium-icon.png" alt="soclai media icons"></li>
        </ul>
-
+        </div>
     </section>
     <section class="recent-work">
         <h2>My recent works</h2> 
 
         <hr>
-        
+        <div class="recent-blog">
```

Comment:
This section of your application is not matching the Figma template of your choosing. Kindly follow the exact layout of the Figma design.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/186837309-813e3035-1d48-44de-8634-6f9808945ec3.png) | ![image](https://user-images.githubusercontent.com/33268927/186837007-ed2528c1-a384-430c-ab4a-a7a54045e525.png) |